generator.rb

```diff
@@ -0,0 +1,61 @@
+
+#refocus to work around index numbers 
+
+
+#Make a method named run_generator. This method has no parameters.
+#find last instance of vowel
+#keep every letter from start to before last vowel
+
+
+def is_vowel?(letter)
+  if ["a", "e", "i", "o", "u"].include?(letter)
+    return true
+  else
+    return false
+  end
+end
+
+def find_vowel(word)
+  part1 = nil
+  word.each_char do |letter|
+    if is_vowel?(letter) == true
+      first_half = word.split(letter)
+      part1 = first_half.first
+    end
+  end
+  return part1
+end
+
+def run_generator()
+  puts "What is the first word to combine?"
+  first = gets.chomp
+  if first.length < 2
+    puts "Invalid input. Please enter a word that is at least two characters long."
+    first = gets.chomp
+  end
+  puts "What is the second word to combine?"
+  second = gets.chomp
+  if second.length < 2
+    puts "Invalid input. Please enter a word that is at least two characters long."
+    second = gets.chomp
```

Comment:
Your code to get the second word is almost exactly the same as the code to get the first word. Could you DRY this up using another method?